paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -709,15 +672,17 @@ internal abstract class BaseSheetViewModel(
         }
     }
 
-    fun onFormFieldValuesChanged(formValues: FormFieldValues?, selectedPaymentMethodCode: String) {
-        paymentMethodMetadata.value?.let { paymentMethodMetadata ->
-            val newSelection = formValues?.transformToPaymentSelection(
-                context = getApplication(),
-                paymentMethod = supportedPaymentMethodForCode(selectedPaymentMethodCode),
-                paymentMethodMetadata = paymentMethodMetadata,
-            )
-            updateSelection(newSelection)
-        }
+    fun createFormHelper(): FormHelper {
```

Comment:
instead of this, should we have a constructor in FormHelper that takes a BaseSheetViewModel? I think that's more consistent with what we do elsewhere